site/src/components/WorkspaceActions/WorkspaceActions.tsx

```diff
@@ -12,14 +12,14 @@ import { Stack } from "../Stack/Stack"
 import { WorkspaceActionButton } from "../WorkspaceActionButton/WorkspaceActionButton"
 
 export const Language = {
-  stop: "Stop workspace",
-  stopping: "Stopping workspace",
-  start: "Start workspace",
-  starting: "Starting workspace",
-  delete: "Delete workspace",
-  deleting: "Deleting workspace",
-  cancel: "Cancel action",
-  update: "Update workspace",
+  stop: "Stop",
+  stopping: "Stopping",
+  start: "Start",
+  starting: "Starting",
+  delete: "Delete",
+  deleting: "Deleting",
+  cancel: "Cancel",
```

Comment:
I would still say "Cancel action" here because it's the one case where the object is not the workspace. Open to thoughts!

Reply:
Good call - this was a mistake!